### PR TITLE
Add set_frame and relative_set_position command parse logic

### LINKED ISSUE #3: Improving `set_position` command
Currently one can set the position of each display in seconds. This is great even because it allows for floating point number precision of milliseconds even. However, there are some situation where this offset would be better described in terms of frames. Additionally, being able to set relative position, as in from 0.0 (start) to 1.0 (end) might also be helpful when the user simply doesn't know the length of each file. Current command value parsing [line 66 67](https://github.com/omarcostahamido/PVM/blob/700b72b6a9ea928fb8d65e426845d87cfc5f9db4/pvm.py#L66-L67) is retrieving only one value, when in fact it could already retrieve a list of values (the rest of the message). Could this be a good opportunity to explore this space?

- [ ] is current `set_position` command enough, or do we need to create new commands for the new features?
- [ ] allow setting position by frame
- [ ] allow setting position relatively (0.0-1.0)

### PR DESCRIPTION
**Pull request by KaneBetter**
TODO: add set_frame and relative_set_position buttons in MAX.
This fixes #3 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ optional arguments:
 ```
 
 On the control machine first edit the [max-init.txt](https://github.com/omarcostahamido/PVM/blob/main/max-init.txt) file. For each Pi device add a numbered line with the video filename, ip, and port. 
-By default, we use port 8001 to receive UDP for our main display. 8002 is used for additional displays. 
+Please use port 8001 to control the first display, and use other ports for the second display.
```
Comment (omarcostahamido) [inline README.md]:
```suggestion
Please use port 8001 to control the first display, and use any other port for the second display. 
```

File: pvm.py [hunk 2]
```diff
@@ -102,32 +102,42 @@ def parse_commands(*args):
...
+		elif command == "set_position":
 			OMX.set_position(float(value))
 			_logger.info("%s command success.", command)
-		elif command=="set_rate":
+		elif command == "set_frame":
+			fps, _ = get_info()
+			frame_pos = 1.0 / fps * float(value)
+			OMX.set_position(frame_pos)
+			_logger.info("%s command success.", command)
+		elif command == "relative_set_position":
```
Comment (omarcostahamido) [inline pvm.py]:
```suggestion
		elif command == "set_relative_position":
```
?

File: README.md (hunk 1 above)
Comment (KaneBetter) [reply README.md]: Done

**Review by omarcostahamido — APPROVED** [review]

**Comment by omarcostahamido** [comment]
do you want to merge #53 into here first?

**Comment by KaneBetter** [comment]
Yes, let me merge it first.

**Comment by KaneBetter** [comment]
Test set_position, set_sposition and set_fposition done.

**Pull request merged**